### ISSUE TITLE
RTL Dropdown: Dropdown icon and list shifts upon opening

### ISSUE THREAD

**Issue by kyfloresca** [infor-design/enterprise-ng#1640]
**Describe the bug**
On RTL, dropdown list moves down and icon moves to the left.

**To Reproduce**
1. Change to RTL
2. Click on any dropdown
3. Open dropdown

**Expected behavior**
The dropdown icon and list should not move.

**Version**
- ids-enterprise-ng: 4.92.2

**Screenshots**
![image](https://github.com/infor-design/enterprise-ng/assets/116287798/8be187ab-4337-41d6-92df-6124b06af894)

**Recording**: https://github.com/infor-design/enterprise-ng/assets/116287798/e425c3bd-bc8a-43dc-95c6-86eca80162c6

**Platform**
 - Windows 11
 - Chrome Version 121.0.6167.185 (Official Build) (64-bit)

**Additional context**
Issue seen in Infor M3.

**Comment by tmcconechy** [comment]
@kyfloresca i think this is duplicate of https://github.com/infor-design/enterprise/pull/8455 
Issue: https://github.com/infor-design/enterprise/issues/8305

Seem like the same thing?

**Comment by kyfloresca** [comment]
@tmcconechy , yep its the same with https://github.com/infor-design/enterprise/pull/8455.

i was double checking the ltr view and i found that the dropdown list also moves slightly down:
![image](https://github.com/infor-design/enterprise-ng/assets/116287798/05662d5c-d9ef-49c2-bfb3-d652fb80048c)

chrome setting: zoom level 125%

is this also handled in https://github.com/infor-design/enterprise/pull/8455? thanks!
